dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -135,22 +135,22 @@ await foreach (StreamingChoice choice in streamingChatCompletions.GetChoicesStre
     /// Generate a new chat message
     /// </summary>
     /// <param name="chat">Chat history</param>
-    /// <param name="requestSettings">AI request settings</param>
+    /// <param name="chatSettings">AI request settings</param>
     /// <param name="cancellationToken">Async cancellation token</param>
     /// <returns>Generated chat message in string format</returns>
-    private protected async Task<string> InternalGenerateChatMessageAsync(
+    private protected async Task<IReadOnlyList<IChatResult>> InternalGetChatResultsAsync(
         ChatHistory chat,
-        ChatRequestSettings requestSettings,
+        ChatRequestSettings? chatSettings,
         CancellationToken cancellationToken = default)
     {
         Verify.NotNull(chat);
-        Verify.NotNull(requestSettings);
+        chatSettings ??= new ChatRequestSettings();
```

Comment:
Small simplification as in other places 😄 
```suggestion
        chatSettings ??= new();
```